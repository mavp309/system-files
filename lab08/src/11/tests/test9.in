Diya
20
34567892
32
