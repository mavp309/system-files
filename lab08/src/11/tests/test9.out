Name: Diya
Age: 20
Roll: 34667892
Marks: 32
